<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendToMainWindow</class>
 <widget class="QMainWindow" name="SendToMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>377</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>377</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>377</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SendToMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="sendToLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>41</y>
      <width>160</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Destination:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSendTo">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>310</y>
      <width>69</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send To</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCancel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>74</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QTreeView" name="folderTree">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>61</y>
      <width>370</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>120</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Send To</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>280</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
